### PR TITLE
Update devices.py

### PR DESCRIPTION
**Pull request by annie-payseur**
Small help text updates for clarity.

### CODE REVIEW

**Review by unparalleled-js — APPROVED** [review]
LGTM

File: src/code42cli/cmds/devices.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def _get_device_info(sdk, device_guid):
     required=False,
     type=str,
     default=None,
-    help="Limit devices to only the ones in the org you specify. "
-    "Note that child orgs will be included.",
+    help="Limit devices to only those in the organization you specify. "
```
Comment (kiran-chaudhary) [inline src/code42cli/cmds/devices.py:211]: Extra space after the period.

**Review by kiran-chaudhary — APPROVED** [review]
Just one minor concern.

File: src/code42cli/cmds/devices.py (hunk 1 above)
Comment (unparalleled-js) [reply src/code42cli/cmds/devices.py:211]: This is a multiline string.  The space at the end is because it's two sentences

**Pull request merged**